CHANGELOG.md

```diff
@@ -12,6 +12,22 @@
 - Formatter uses a compact, single-line representation for array and message literals
   in option values that are sufficiently simple (single scalar element or field).
 - Include `LICENSE` file in the module on `buf push`.
+- New compiler that is XX% faster, uses XX% less memory, and addresses a few bugs
```

Comment:
Make sure we update before merging

Reply:
@bufdev, you could "request changes" on the PR, and that will block merging. (Or some other reviewer could do that.)

Reply:
Updated with more details and actual stats. Related: https://github.com/bufbuild/protocompile/pull/64